tests/input.spec.ts

```diff
@@ -347,34 +342,30 @@ test('With error state, displays message below input box in red', async ({
 test('Given type number, only dispatches valid and new number values', async ({
   page,
 }) => {
-  const inputNumber = await page.getByTestId('input-number');
-  const input = await inputNumber.locator('input').first();
+  const inputNumber = page.getByTestId('input-number');
+  const input = inputNumber.locator('input').first();
 
-  const noInputEvent1 = await waitForCustomEventTimeout(page, 'input');
-  await input.fill('NaN');
-  await noInputEvent1;
+  // TODO(mc, 2023-03-24): `NaN` is a valid number input in JS
```

Comment:
One test failure noted in PR description. Anyone have any thoughts here?

Reply:
I realized this is firing an event because the value goes from a valid starting number (set as a parameter in `input-test.vue`) to a blank string (the component rejects the characters "NaN")

I have a fix for this, maybe leave a TODO comment here and I can merge my solution after